service/hook/gogs/gogs.go

```diff
@@ -82,17 +82,39 @@ func transformCodePushEvent(codePushEvent CodePushEventModel) hookCommon.Transfo
 
 	if !isLastCommitFound {
 		return hookCommon.TransformResultModel{
-			Error: errors.New("The commit specified by 'after' was not included in the 'commits' array - no match found"),
+			Error:      errors.New("The commit specified by 'after' was not included in the 'commits' array - no match found"),
+			ShouldSkip: true,
```

Comment:
why did you change this to `ShouldSkip: true`?

This will answer with a "200" response, not with an error one.